### DISCUSSION TITLE
Category wise separation on website?

### DISCUSSION THREAD

**Discussion by Anuragtech02** [IET-DAVV/website-client#60] [Q&A]
Asking this as a question
Check out the current website at [ietdavv.edu.in](https://ietdavv.edu.in).
## Shall we categorize content on our website into three categories (Students | Teachers | Visitors)?

Something like this
![categorization](https://user-images.githubusercontent.com/55744578/204184836-1ed86fdb-3663-4af7-b483-c629fcbd2140.png)

As per our analysis, there would be some pros and cons to this.
**Pros**
- It will be easier for users to find the information they're looking for
- It will reduce clutter on the landing page and decision-making would be easier

**Cons**
- Would create problems in ranking the website (SEO) due to lesser information on the homepage
- Will increase the number of clicks for a visitor on the website
- A visitor might not be aware or educated enough to choose a category for herself/himself

We need suggestions from the community and others on whether to go with category-wise separation of content or not. 

> **_NOTE:_** Kindly mention Pros & Cons in your comment. Thanks.

**Comment by rishigupta1109** [comment]
I think since it is affecting the SEO we should not go for it. As one of the pros of our current website is that SEO.  

But the partition will definately ease the search and decrease the clutter.

Conclusively if in gg search website comes on top I suggest we can go for partition else I don't think it would be a great idea to compromise SEO.

**Reply by Anuragtech02** [reply]
It is not completely compromising SEO, it is just reducing content so for crawlers, it would be slightly difficult to rank us. Not completely changing anything.

**Reply by rishigupta1109** [reply]
Can we add the content somewhere which is hidden to us but not to crawler?

**Reply by Anuragtech02** [reply]
We can, but SEO depends on a lot of things, like the way the content is presented
For example: An `h1` heading makes much more impact to the SEO than the content in `p` tag
And things that are visible on the screen also are preferred.

**Comment by Aditya-Raghuvanshi** [comment]
You may categorise the students and faculty and for  the visitors you can display all the relevant operations on the home page itself
Below student and techer option
Here option of Student and techer are in the same row
This will also increase the content as well as quite effective,Then previous one.

**Reply by Anuragtech02** [reply]
Can you elaborate more? What things can we sort into student and faculty categories?

**Reply by Aditya-Raghuvanshi** [reply]
I mean to say, can we add only students and faculty links on the homepage so that they will go for their information in their respective section , and for visitors, we may display subcategories on the landing home page , by this number of clicks for visitors will also decrease.

**Comment by Mohit-Aasirwal** [comment]
I don't think that faculty section will contain as much info as we think. imo It is better to divide **branch** wise pages for separation of content, and then there should be main college content on landing page. In this way, SEO can be preserved and relevant info can be separated.
Naturally user will go to their respective pages after visiting home page of website.
